hwdb.d/60-keyboard.hwdb

```diff
@@ -1645,6 +1645,15 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnOQO*Inc.*:pnOQO*Model*2*:*
  KEYBOARD_KEY_f2=volumedown
  KEYBOARD_KEY_f3=volumeup
 
+###########################################################
+# OrangePi
+###########################################################
+
+# NEO
+evdev:name:AT Translated Set 2 keyboard:dmi:*:svnOrangePi:pnNEO-01:*
+ KEYBOARD_KEY_66=f15
+ KEYBOARD_KEY_67=f16
```

Comment:
Could you add a brief comment for each key code? That is, please mention `LC` and `RC`.

Reply:
Added the comments. After double checking the LC/RC keys properly scan, its the Home and Gamepad buttons that do not. I've updated my original comment.